homeassistant/components/sensor/yr.py

```diff
@@ -185,7 +187,8 @@ def try_again(err: str):
                 model = self.data['meta']['model']
                 if '@nextrun' not in model:
                     model = model[0]
-                self._nextrun = dt_util.parse_datetime(model['@nextrun'])
+                self._nextrun = dt_util.parse_datetime(model['@nextrun']) +\
```

Comment:
I removed the 10 minutes delay.
So we will ask for new data 1 to 11 minutes after "nextrun"